packages/ng-schematics/package.json

```diff
@@ -8,7 +8,8 @@
     "dev:test": "npm run test --watch",
     "dev": "npm run build --watch",
     "test": "wireit",
-    "sandbox": "node tools/sandbox.js"
+    "sandbox": "node tools/sandbox.js",
+    "sandbox:test": "node tools/sandbox.js --test"
```

Comment:
test:sandbox

Reply:
That is not correct, we are not testing the sandbox we are running the `schematics:test` in the sandbox.